response_operations_ui/views/collection_exercise.py

```diff
@@ -162,6 +163,16 @@ def _validate_collection_instrument():
                 "header": "Error: wrong file type for Collection instrument",
```

Comment:
🎨 As we're modifying this file, can we title or lower case "Collection Instrument"  (consistency would be lower)